src/react-queries/hooks/useAllowance.ts

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from '@tanstack/react-query'
+import { reactQueries } from 'react-queries'
+
+import { selectAllowanceCryptoBaseUnit } from './selectors'
+
+export const useAllowance = ({
+  assetId,
+  spender,
+  from,
+}: {
+  assetId: string | undefined
+  spender: string | undefined
+  from: string | undefined
+}) => {
```

Comment:
nitpick type and easier to read one line function definition. ps. did I miss something where we are using `string | undefined` over `?: string` these days?
```suggestion
type UseAllowanceArgs = {
  assetId?: string
  spender?: string
  from?: string
}

export const useAllowance = ({ assetId, spender, from }: UseAllowanceArgs) => {
```

Reply:
These are required, but *may* be undefined depending on the render cycle we're in.
Since this is effectively a wrapper over the `allowanceCryptoBaseUnit` query, which has an `enabled` guard, this is effectively ensuring that these are always passed to the hook/query, and also ensuring types convey the expected shape.

Moved from inline to actual type in 1f9d1f9a853ccfc2bbf8d19dce2525901d17d880

Reply:
Got it, thanks!